use qurious::error::Result;

fn main() -> Result<()> {
    Ok(())
}
